pkg/csi_driver/controller.go

```diff
@@ -101,6 +104,30 @@ func (s *controllerServer) CreateVolume(ctx context.Context, req *csi.CreateVolu
 			return nil, status.Error(codes.AlreadyExists, err.Error())
 		}
 	} else {
+
+		// Get the reserved-IPV4-cidr from the request. If value is not provided, reservedIPV4Range is defaulted to ""
```

Comment:
Can you update all the comments?  In this particular comment, it's referring to variables that don't exist. many of the other comments are out of date too